File-Formats/DocIO/Working-with-Find-and-Replace.md

```diff
@@ -799,6 +799,116 @@ Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Replace.docx", "applic
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Find-and-Replace/Find-and-replace-paragraphs-with-text).
 
+## Find next and replace with formatted text
+
+The following code example illustrates how to replace the text extended to several paragraphs with particular text in the document:
```

Comment:
The following code example illustrates how to replace the text extended to several paragraphs with a particular text in the document.

Reply:
@Chrispineagunja -Done